tests/puppeteer/campaigns/functional/BO/catalog/brandsAndSuppliers/suppliers/02_filterQuickEditAndBulkActionsSupplier.js

```diff
@@ -0,0 +1,285 @@
+      await expect(numberOfSuppliersAfterReset).to.be.equal(numberOfSuppliers);
+    });
+
+    it('should filter supplier by enabled', async function () {
+      await this.pageObjects.suppliersPage.filterSupplierEnabled(firstSupplierData.enabled);
+      // Check number od suppliers
+      const numberOfSuppliersAfterFilter = await this.pageObjects.suppliersPage.getNumberFromText(
+        this.pageObjects.suppliersPage.gridHeaderTitle,
+      );
+      await expect(numberOfSuppliersAfterFilter).to.be.at.most(numberOfSuppliers);
+      // check text column of first row after filter
+      const textColumn = await this.pageObjects.suppliersPage.getTextContent(
+        this.pageObjects.suppliersPage.tableColumn
+          .replace('%ROW', 1)
+          .replace('%COLUMN', 'active'),
+      );
+      await expect(textColumn).to.contains('check');
+    });
+
+    it('should reset filter', async function () {
+      const numberOfSuppliersAfterReset = await this.pageObjects.suppliersPage.resetAndGetNumberOfLines();
+      await expect(numberOfSuppliersAfterReset).to.be.equal(numberOfSuppliers);
+    });
+  });
+  // 3: Quick Edit Suppliers
+  describe('Quick edit first supplier', async () => {
+    it('should filter supplier by name', async function () {
+      await this.pageObjects.suppliersPage.filterTable(
+        'input',
+        'name',
+        firstSupplierData.name,
+      );
+      // Check number od suppliers
+      const numberOfSuppliersAfterFilter = await this.pageObjects.suppliersPage.getNumberFromText(
+        this.pageObjects.suppliersPage.gridHeaderTitle,
+      );
+      await expect(numberOfSuppliersAfterFilter).to.be.at.most(numberOfSuppliers);
+      // check text column of first row after filter
+      const textColumn = await this.pageObjects.suppliersPage.getTextContent(
+        this.pageObjects.suppliersPage.tableColumn
+          .replace('%ROW', 1)
+          .replace('%COLUMN', 'name'),
+      );
+      await expect(textColumn).to.contains(firstSupplierData.name);
+    });
+
+    it('should disable first supplier', async function () {
+      const isActionPerformed = await this.pageObjects.suppliersPage.updateEnabledValue(
+        '1',
+        false,
+      );
+      if (isActionPerformed) {
+        const resultMessage = await this.pageObjects.suppliersPage.getTextContent(
+          this.pageObjects.suppliersPage.alertSuccessBlockParagraph,
+        );
+        await expect(resultMessage).to.contains(this.pageObjects.suppliersPage.successfulUpdateStatusMessage);
+      }
+      const isStatusChanged = await this.pageObjects.suppliersPage.elementVisible(
+        this.pageObjects.suppliersPage.enableColumnNotValidIcon.replace('%ROW', 1),
+        100,
+      );
+      await expect(isStatusChanged).to.be.true;
+    });
+
+    it('should enable first supplier', async function () {
+      const isActionPerformed = await this.pageObjects.suppliersPage.updateEnabledValue(
```

Comment:
Maybe it is better to delete the variables in parameter because we use the default variables in the function

Reply:
Maybe it's the default value, but don't you think it's more readable for reviewers if we indicate that value in tests ?

Reply:
Yes you are right, it's more readable for reviewers. 
But, what's the impact of the default values here ?
You can add a comment before the function to explicate that you use the dafault values

Reply:
in second thought, I can't for this one, since the second param must be edited. but you are right for others.